src/routes/wallet/atomic-swap/tx-proposal.routes.js

```diff
@@ -42,6 +44,12 @@ txProposalRouter.get(
   getLockedUTXOs,
 );
 
+txProposalRouter.post(
+  '/fetch-from-service',
```

Comment:
I don't think I like the name of this API. Only `fetch` would be enough for me, what do you think?

Reply:
I think it's a good name. Refactored it on [4bdc332](https://github.com/HathorNetwork/hathor-wallet-headless/pull/277/commits/4bdc3327fa7aaafd1c4d78bae9b058de7d775068)